spec/plugin_spec.rb

```diff
@@ -15,4 +15,15 @@
     expect(post.post_uploads.size).to eq(1)
     expect(post.post_uploads.first.upload).to eq(upload)
   end
+
+  it 'cannot enable encrypt if unsafe CSP' do
```

Comment:
Can you add a test case with a valid list of CSP directives ('self' and/or some URLs)? Just to make sure this doesn't regress in the future.